docs/application/native/guides/connectivity/vpn.md

```diff
@@ -213,7 +213,7 @@ To set the interface parameters:
 
 To configure the interface and connect to the VPN service through an application control:
 
-1. Define a function to launch the [VPN application control](../app-management/common-appcontrol.md#vpnservice) and a callback to handle the results:
+1. Define a function to launch the [VPN application control](../app-management/common-appcontrols.md#vpnservice) and a callback to handle the results:
```

Comment:
It looks like the \<a name\> tags are removed, and we are using links generated by \# only.
`vpnservice` is old tag and removed now. To jump to the right location, it should be `vpn-service`.